backend/src/routes/project_status.js

```diff
@@ -1,20 +1,32 @@
 const controller = require("../controllers/project_status");
 const validators = require("../validators/project_status");
-const what = "project_status";
+const what = "status";
 
 const routes = [
   {
     method: "GET",
-    url: `/${what}`,
+    url: `/projects/:id/${what}`,
     schema: validators.getAll,
-    handler: controller.getAll,
+    handler: controller.findAllByProject,
   },
   {
     method: "GET",
-    url: `/${what}/:id`,
+    url: `/projects/${what}/:id`,
     schema: validators.getOne,
     handler: controller.getOne,
   },
+  {
+    method: "PUT",
+    url: `/projects/${what}/:id`,
+    schema: validators.updateOne,
+    handler: controller.updateOne,
+  },
+  {
+    method: "POST",
+    url: `/projects/${what}`,
+    // schema: validators.addOne,
+    handler: controller.addOne,
+  },
```

Comment:
We're missing tests for the add/update routes. They should be pretty straightforward.